### ISSUE TITLE
9. All Possible Full Binary Trees(TREES)

### ISSUE THREAD

**Issue by SnehaAgarwal361** [LetsGrowMoreCommunity/DSA-Playyard#190]
# Contribution Guidelines for this question-

- Languages available: C++, Java, Python
- Well Commented Code
- In the starting, add a brief explanation of the algorithm
- if possible, add time and space complexity
- kindly upload your code in TREES -> Language (C++, Java, Python) Folder
- NO PLAGIARISM

Also, watch this video and the README section before contributing to the project.
https://youtu.be/qXeaN_ZVVIc

**Comment by soma2000-lang** [comment]
@SnehaAgarwal361  PLease assign me this issue.I will code in C++.I am a LGM SOC partucipant

**Comment by SnehaAgarwal361** [comment]
Yes you can start working on the issue, please do take care of the contribution guidelines!
